.circleci/config.yml

```diff
@@ -137,8 +137,8 @@ jobs:
     steps:
       - setup_vm
       - run:
-          name: 'Install Protobuf'
-          command: pip3 install --user protobuf
+          name: 'Install Validator Dependencies'
+          command: sh ./.circleci/install_validator_dependencies.sh
```

Comment:
`sh` is not platform independent, and should be removed. I think the problem is that the script you added is not executable. You might need to `chmod 755` it.

Reply:
Done